# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/websockets/constructor.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/websockets/constructor.any.js.ini
--- a/testing/web-platform/meta/websockets/constructor.any.js.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/websockets/constructor.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -4,22 +4,26 @@
 
 [constructor.any.worker.html?wpt_flags=h2]
   expected:
-    if (os == "win") and (processor == "x86") and debug: OK
     if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
     if (os == "android") and debug and not webrender: ["TIMEOUT", "OK", "ERROR"]
-    if (os == "android") and not debug and webrender: ["TIMEOUT", "ERROR"]
-    if (os == "win") and (processor == "x86_64"): OK
-    TIMEOUT
+    if (os == "android") and debug and webrender: ["TIMEOUT", "OK", "CRASH"]
+    if (os == "linux") and (processor == "x86_64"): TIMEOUT
+    if (os == "linux") and (processor == "x86"): ["TIMEOUT", "ERROR"]
+    if (os == "android") and not debug: ["TIMEOUT", "ERROR"]
 
 [constructor.any.worker.html?wss]
 
 [constructor.any.html?wpt_flags=h2]
   expected:
+    if (os == "linux") and not fission and debug and webrender and not swgl: ["TIMEOUT", "CRASH"]
+    if (os == "linux") and not fission and debug and webrender and swgl: TIMEOUT
+    if (os == "linux") and not fission and debug and not webrender: TIMEOUT
     if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if (os == "win") and (processor == "x86") and debug: OK
-    if (os == "android") and debug and not webrender: ["TIMEOUT", "ERROR"]
-    if (os == "android") and not debug and webrender: ["TIMEOUT", "ERROR"]
-    if (os == "win") and (processor == "x86_64"): OK
-    TIMEOUT
+    if (os == "linux") and not fission and not debug: TIMEOUT
+    if (os == "linux") and fission and not debug: ["TIMEOUT", "ERROR"]
+    if (os == "linux") and fission and debug: TIMEOUT
+    if (os == "android") and debug and webrender: ["TIMEOUT", "ERROR"]
+    if (os == "android") and debug and not webrender: ["TIMEOUT", "ERROR", "CRASH"]
+    if (os == "android") and not debug: ["TIMEOUT", "ERROR"]
 
 [constructor.any.html]